tests/test_models.py

```diff
@@ -460,3 +483,62 @@ class TestModel(models.Model):
     __test__ = False  # this is not a test
     value = models.IntegerField(null=True)
     other = models.IntegerField(null=True)
+
+
+class TestAuditEventBootstrapping(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.aerodrome_details = {
+            "KIAD": 313,
+            "VIDP": 777,
+            "FACT": 151,
+        }
+        for icao, amsl in cls.aerodrome_details.items():
+            Aerodrome.objects.create(
+                icao=icao,
+                elevation_amsl=amsl,
+                amsl_unit="ft",
+            )
+
+    def test_bootstrap_existing_model_records(self):
+        with patch.object(AuditEvent.objects, "bulk_create",
+                          side_effect=AuditEvent.objects.bulk_create) as mock:
+            self._verify_bootstrap_existing_model_records(None)
+            mock.assert_called_once_with(ANY)
+
+    def test_bootstrap_existing_model_records_batched(self):
+        with patch.object(AuditEvent.objects, "bulk_create",
+                          side_effect=AuditEvent.objects.bulk_create) as mock:
+            self._verify_bootstrap_existing_model_records(1)
+            self.assertEqual(len(self.aerodrome_details), mock.call_count)
+            mock.assert_called_with(ANY, batch_size=1)
+
```

Comment:
Could you add a test case for running `bootstrap_existing_model_records` on a model with no existing instances?

Reply:
Added in a555f68.